### PR TITLE
[Terra-SearchSelect] Included the "Invalid" property in the example of an invalid case and set it to true.

### PR DESCRIPTION
**Pull request by AH106586Harika**
### Summary
Fixed - Set the aria-invalid attribute to true for error input fields and false when resolving errors.

**What was changed:**
Included the "Invalid" property in the example of an invalid case and set it to true.
Introduced relevant visual label for search select example.

**Why it was changed:**
To declare as "Invalid" through the screen reader, indicating a field error.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

[UXPLATFORM-9848](https://jira2.cerner.com/browse/UXPLATFORM-9848)
[UXPLATFORM-9840](https://jira2.cerner.com/browse/UXPLATFORM-9840)

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-form-select/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Added
+  * Added 'aria-invalid' attribute to set to true for error input fields and false when resolving errors.
```
Comment (sdadn) [inline packages/terra-form-select/CHANGELOG.md:5]:
```suggestion
  * Added 'aria-invalid' attribute which will be set to true for error input fields and false when resolving errors. 
```

Comment (AH106586Harika) [reply packages/terra-form-select/CHANGELOG.md]: Done.

**Review by sugan2416 — APPROVED** [review]

File: packages/terra-core-docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## Unreleased
+* Added
+  * Added visual label and associated with input field using 'for' and 'id'.
```
Comment (supreethmr) [inline packages/terra-core-docs/CHANGELOG.md:5]:
```suggestion

* Added
  * Added visual label and associated with input field using 'for' and 'id'.
```
Need a line break to keep it consistent with other logs

Comment (AH106586Harika) [reply packages/terra-core-docs/CHANGELOG.md:6]: Done.

**Review by supreethmr — APPROVED** [review]

**Pull request merged**